content/en/monitors/status/graphs.md

```diff
@@ -66,6 +66,13 @@ To filter the view for an individual group, hover over the group title and click
 
 When investigating past status changes, the color dot next to the group title indicates the group's current status.
 
+### Change Tracking
+The Change Tracking graph allows you to view and analyze changes related to your service and its dependencies that occurred around the same time as the alert, as such events are often the root cause of problems.
+
+{{< img src="/monitors/status/change_tracking_monitor_status_page.png" alt="Example of a deployment shown in the monitor status page" style="width:100%;" >}}
+
+Change Tracking supports multiple changes like Deployments, Feature Flags or Database Modifications. To see the full list and setup requirements, see the Change Tracking [documentation][2].
```

Comment:
Are these deployment and feature flags product features or types of changes. Recommend lower case if they're types of changes that we graph.

```suggestion
Change Tracking supports multiple changes like deployments, feature flags or database modifications. To see the full list and setup requirements, see the [Change Tracking][2] documentation.
```